dos-donts/src/index.stories.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
```

Comment:
do we need useEffect ?

Reply:
No, I'll remove it. It would be nice to find a way to have relevant TS errors locally. Because right there are lots of errors due to imports